packages/webpack-cli/src/webpack-cli.ts

```diff
@@ -976,6 +976,17 @@ class WebpackCLI implements IWebpackCLI {
         description: "Stop watching when stdin stream has ended.",
         negatedDescription: "Do not stop watching when stdin stream has ended.",
       },
+      {
+        name: "fail-on-warnings",
+        configs: [
+          {
+            type: "enum",
+            values: [false],
```

Comment:
Sorry, typo in my earlier suggestion. it should be true.
```suggestion
            values: [true],
```